emsdk.py

```diff
@@ -1332,6 +1332,16 @@ def emscripten_npm_install(tool, directory):
       return False
 
   print('Done running: npm ci')
+
```

Comment:
Do we still all the code above for patching closure compiler.   

The comment says "delete the Java version, since that takes up ~12.5MB of installation space that is no longer needed.".  Is it really worth the extra complexity to save 12.5mb locally?   Or are you/we shipping the resulting local emsdk installation?  Even so, is 12.5mb worth it?

Reply:
We are shipping the resulting emsdk installation.

I think it is worth the save, given that it is "easy" dead code. Our WebGL support module gets downloaded by thousands and thousands of developers, it all accumulates up in CDN costs.

Reply:
SGTM.